jira/client.py

```diff
@@ -879,13 +881,19 @@ def add_attachment(self, issue, attachment, filename=None):
 
         if "MultipartEncoder" not in globals():
             method = "old"
-            r = self._session.post(
-                url,
-                files={"file": (fname, attachment, "application/octet-stream")},
-                headers=CaseInsensitiveDict(
-                    {"content-type": None, "X-Atlassian-Token": "nocheck"}
-                ),
-            )
+            try:
+                r = self._session.post(
+                    url,
+                    files={"file": (fname, attachment, "application/octet-stream")},
+                    headers=CaseInsensitiveDict(
+                        {"content-type": None, "X-Atlassian-Token": "nocheck"}
+                    ),
+                )
+            except Exception as e:
+                raise e
```

Comment:
if you raise e here it will not show the proper traceback
```suggestion
            except Exception as e:
                raise
```
or better maybe even dont catch exceptions
```suggestion
```